server/src/services/UserService.ts

```diff
@@ -0,0 +1,95 @@
+import { sign } from 'jsonwebtoken';
+import { Context } from '../context';
+import { NexusGenRootTypes } from '../generated/nexus';
+import { ErrorEmailForUserExists, ErrorString } from '../utils/error';
+import { AuthType, Gender } from '../types/models';
+import { USER_SIGNED_IN } from '../types/resolvers';
+
+export interface SocialUserInput {
+  socialId: string;
+  authType: AuthType;
+  name: string;
+  email: string;
+  birthday?: Date;
+  gender?: Gender;
+  phone?: string;
+}
+
+export class UserService {
```

Comment:
Why not exporting this as a `default`?
Also, we are more likely to make `typescript` with functional programming.
Could you kindly tell me why the class would be better in this case?

Reply:
> Why not exporting this as a default?

Actually, this was my preference. Using named exports forces others to use same name, and I think it will make code easier to read and more predictible.

> we are more likely to make typescript with functional programming.

I thought using static methods and using functions are not that different. But if coding guide in this project are using functions, I will follow 🙂

Reply:
I think it'd be better to export as default if there are only one main logic exports (others are types). However, if there are more to come to be exported, it is better to export them separately as you've mentioned.